website/versioned_docs/version-0.70/running-on-simulator-ios.md

```diff
@@ -9,13 +9,13 @@ Once you have your React Native project initialized, you can run `npx react-nati
 
 ## Specifying a device
 
-You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 13"`. If you wish to run your app on an iPhone SE (2nd generation), run `npx react-native run-ios --simulator='iPhone SE (2nd generation)'`.
+You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 14"`. If you wish to run your app on an iPhone SE (3rd generation), run `npx react-native run-ios --simulator='iPhone SE (3rd generation)'`.
```

Comment:
I'm unsure if we can update also the docs for 0.70.
The change seems to be on the CLI 9.1. Is that inside React Native 0.70?

Reply:
yep https://github.com/facebook/react-native/pull/34803

Reply:
Hi @cortinico thanks for the review, recently added see https://github.com/facebook/react-native/pull/34803

Reply:
Yup but this is on `main`, not on 0.70

Reply:
Yup it will go now in the `next` version, which will also become 0.71.0